<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>374</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>车辆管理</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>373</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QToolBox" name="toolBox">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="page_3">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>431</width>
                <height>353</height>
               </rect>
              </property>
              <attribute name="label">
               <string>出售车辆</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout">
               <item row="5" column="0" colspan="2">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>金额</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0" colspan="2">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>品牌</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0" colspan="2">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>价格</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0" colspan="2">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>数量</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1" colspan="2">
                <widget class="QComboBox" name="factory"/>
               </item>
               <item row="3" column="2">
                <widget class="QWidget" name="widget_3" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QLineEdit" name="price"/>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>万</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="6" column="0" colspan="3">
                <widget class="QWidget" name="widget_6" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout_6">
                  <item>
                   <spacer name="horizontalSpacer_3">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>107</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="ok">
                    <property name="text">
                     <string>确定</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="cancel">
                    <property name="text">
                     <string>取消</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_4">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>107</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="4" column="2">
                <widget class="QWidget" name="widget_4" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <item>
                   <widget class="QSpinBox" name="num"/>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_5">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>145</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_7">
                    <property name="text">
                     <string>剩余数量：</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="last">
                    <property name="text">
                     <string>TextLabel</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>厂商</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="2">
                <widget class="QWidget" name="widget_5" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <widget class="QLineEdit" name="sum_price"/>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_10">
                    <property name="text">
                     <string>万</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QComboBox" name="brand"/>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="page_4">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>425</width>
                <height>353</height>
               </rect>
              </property>
              <attribute name="label">
               <string>新车入库</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QTableView" name="tableView"/>
               </item>
               <item>
                <widget class="QWidget" name="widget_7" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <item>
                   <spacer name="horizontalSpacer_6">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>103</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="commit">
                    <property name="text">
                     <string>提交</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="revert">
                    <property name="text">
                     <string>取消</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_7">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>102</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="listWidget"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QWidget" name="widget_8" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QWidget" name="widget_9" native="true">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>66</height>
              </size>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>厂商</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="changshang"/>
              </item>
              <item>
               <spacer name="horizontalSpacer_8">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>267</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QWidget" name="widget_10" native="true">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>66</height>
                 </size>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_9">
                 <item>
                  <spacer name="horizontalSpacer_9">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>150</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_11">
                   <property name="text">
                    <string>厂商品牌</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_10">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>150</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="Histogram" name="wg_histogram" native="true"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_12" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QWidget" name="widget_13" native="true">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>66</height>
              </size>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>品牌</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="pinpai"/>
              </item>
              <item>
               <spacer name="horizontalSpacer_11">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>47</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_2">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QWidget" name="widget_14" native="true">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>66</height>
                 </size>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_12">
                 <item>
                  <spacer name="horizontalSpacer_12">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_13">
                   <property name="text">
                    <string>品牌车辆</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_13">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="Pie" name="wg_pie" native="true"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>909</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="sellManager">
    <property name="title">
     <string>销售管理</string>
    </property>
    <addaction name="carManager"/>
    <addaction name="sell_count"/>
   </widget>
   <widget class="QMenu" name="sysManager">
    <property name="title">
     <string>系统管理</string>
    </property>
    <addaction name="reset_pwd"/>
    <addaction name="quit"/>
   </widget>
   <addaction name="sellManager"/>
   <addaction name="sysManager"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="carManager">
   <property name="text">
    <string>车辆管理</string>
   </property>
  </action>
  <action name="sell_count">
   <property name="text">
    <string>销售统计</string>
   </property>
  </action>
  <action name="reset_pwd">
   <property name="text">
    <string>修改密码</string>
   </property>
  </action>
  <action name="quit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Histogram</class>
   <extends>QWidget</extends>
   <header>histogram.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Pie</class>
   <extends>QWidget</extends>
   <header location="global">pie.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
